assets/script.js

```diff
@@ -41,75 +47,208 @@ function selectionsValue (e) {
+                        console.log(data);
+                        dataSorter(data);
+                        return;
+                    })
+                }
+            })
+        } else{
+            var imdbAPI = "https://imdb-api.com/API/AdvancedSearch/k_zns86b2w?title=" + userInput;
+            fetch(imdbAPI).then(function (response) {
+                if (response.ok) {
+                    response.json().then(function (data) {
+                        console.log(data);
+                        dataSorter(data);
+                        return;
+                    })
+                }
+            })
+        }
+    }else{
+        if ((tvcheckBox === true) && (filmcheckBox === true)) {
+            var imdbAPI = "https://imdb-api.com/API/AdvancedSearch/k_zns86b2w?title_type=feature,tv_series&genres=" + userInput;
+            fetch(imdbAPI).then(function (response) {
+                if (response.ok) {
+                    response.json().then(function (data) {
+                        console.log(data);
+                        dataSorter(data);
+                        return;
+                    })
+                }
+            })
+        } else if ((tvcheckBox === false) && (filmcheckBox === true)) {
+            var imdbAPI = "https://imdb-api.com/API/AdvancedSearch/k_zns86b2w?title_type=feature&genres=" + userInput;
+            fetch(imdbAPI).then(function (response) {
+                if (response.ok) {
+                    response.json().then(function (data) {
+                        console.log(data);
+                        dataSorter(data);
+                        return;
+                    })
+                }
+            })
+        } else if ((tvcheckBox === true) && (filmcheckBox === false)) {
+            var imdbAPI = "https://imdb-api.com/API/AdvancedSearch/k_zns86b2w?title_type=tv_series&genres=" + userInput;
+            fetch(imdbAPI).then(function (response) {
+                if (response.ok) {
+                    response.json().then(function (data) {
+                        console.log(data);
+                        dataSorter(data);
+                        return;
+                    })
+                }
+            })
+        } else{
+            var imdbAPI = "https://imdb-api.com/API/AdvancedSearch/k_zns86b2w?genres=" + userInput;
+            fetch(imdbAPI).then(function (response) {
+                if (response.ok) {
+                    response.json().then(function (data) {
+                        console.log(data);
+                        dataSorter(data);
+                        return;
+                    })
+                }
+            })
+        }
+    }
+}
+//dataSorter function loops through the data to find the major info and inputs it into seperate arrays and id is saved as a string
+//not sure if this is good or if it would have been better to put all of the info into a single array
+function dataSorter(data){
```

Comment:
Rather thank placing each data value in a different array, I think it would be better to put it in an array of objects. For example:

declare the empty array at the beginning of the search titles function:
var movieObjectArray = []

Then in the loop:

var newMovieObj = {
title: data.results[i].title,
poster: data.results[i].image,
plot: data.results[i].plot,
rating: data.results[i].imDbRating,
id: results[i].id
}

That way all the right data is grouped together, and Zack will be able to loop through the data more easily in his function.